### PR TITLE
Pkg.wdc

### PR DESCRIPTION
**Pull request by rusdevops**
* I have checked that this pull request contains only
  `.travis.yml`/`appveyor.yml` changes. All other changes send
  to https://github.com/ruslo/hunter. **[No]**

* I have checked that no toolchains removed from CI configs, they are commented
  out instead so other developers can enable them back easily and to simplify
  merge conflict resolution. **[Yes]**

* I have checked that for every commented out toolchain there is a link to the
  broken CI build page or to the minimum compiler requirements documentation
  so other developers can figure out what was the problem exactly. **[Yes]**

* I have checked that for every enabled toolchain corresponding package passed
  all stages of update cycle: test/merge/upload/release. **[Yes]**

### CODE REVIEW

**Comment by rusdevops** [comment]
appveyor ✅: https://ci.appveyor.com/project/rusdevops/hunter/builds/21392906
travis ✅: https://travis-ci.org/rusdevops/hunter/builds/475500957

**Comment by rusdevops** [comment]
Updated!

File: cmake/configs/default.cmake [hunk 1]
```diff
@@ -1,314 +1,508 @@
...
+hunter_default_version(SimpleSignal VERSION 0.0.0-79c3f68-p1)
+hunter_default_version(Snappy VERSION 1.1.6-p0)
+hunter_default_version(Sober VERSION 0.1.3)
+hunter_default_version(SuiteSparse VERSION 5.1.2-p0)
+hunter_default_version(TCLAP VERSION 1.2.2-p1)
+hunter_default_version(TIFF VERSION 4.0.2-p5)
+hunter_default_version(Tesseract VERSION 3.05.01-hunter-3)
+hunter_default_version(Urho3D VERSION 1.7-p15)
+hunter_default_version(VulkanMemoryAllocator VERSION 2.1.0-p0)
+hunter_default_version(WDC VERSION 1.1.5)
```
Comment (ruslo) [inline cmake/configs/default.cmake:148]: This change and in particular https://github.com/ingenue/hunter/pull/327/commits/082a56d93cf10338c2dae51af284032cb940cc1b commit don't belong here, please send it to https://github.com/ruslo/hunter

> I have checked that this pull request contains only
> .travis.yml/appveyor.yml changes. All other changes send
> to https://github.com/ruslo/hunter. [**No**]

It's expected that you will answer **Yes** here.

See also: https://docs.hunter.sh/en/latest/creating-new/update.html#add-toolchains

**Review by ruslo — CHANGES_REQUESTED** [review]

**Comment by ruslo** [comment]
Branch `pkg.wdc` updated:
* https://ci.appveyor.com/project/ingenue/hunter/builds/21513601
* https://travis-ci.org/ingenue/hunter/builds/477676055?utm_source=github_status&utm_medium=notification

File: cmake/configs/default.cmake (hunk 1 above)
Comment (rusdevops) [reply cmake/configs/default.cmake:148]: I send pull request: https://github.com/ruslo/hunter/pull/1698

**Pull request closed without merging**